README.md

```diff
@@ -146,7 +146,7 @@ SIG-specific GitHub discussions.
 | <a id="sig-entities"></a>Specification: Entities | Every other Thursday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/15Yt9ss2_EhuFPqItPbk4vjfpeRDAQ5WCUVuY_kCeOAo) | [#otel-entities](https://cloud-native.slack.com/archives/C06QEG97W7L) | [calendar-entities](https://groups.google.com/a/opentelemetry.io/g/calendar-entities) | [Josh Suereth](https://github.com/jsuereth),<br/>[Tigran Najaryan](https://github.com/tigrannajaryan) | [Severin Neumann](https://github.com/svrnm) | 
 | <a id="sig-opamp"></a>OpAMP | Every other Wednesday at 11:00 PT | [Google Doc](https://docs.google.com/document/d/19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4) | [#otel-opamp](https://cloud-native.slack.com/archives/C02J58HR58R) | [calendar-opamp](https://groups.google.com/a/opentelemetry.io/g/calendar-opamp) | [Tigran Najaryan](https://github.com/tigrannajaryan) | [Ted Young](https://github.com/tedsuo) | 
 | <a id="sig-prometheus"></a>Prometheus Interoperability | Every other Wednesday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8) | [#otel-prometheus](https://cloud-native.slack.com/archives/C01LSCJBXDZ) | [calendar-prometheus](https://groups.google.com/a/opentelemetry.io/g/calendar-prometheus) |  | [Alolita Sharma](https://github.com/alolita) | 
-| <a id="sig-faas"></a>Functions as a Service (FAAS) | Every other Wednesday at 8:00 PT | [Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc) | [#otel-faas](https://cloud-native.slack.com/archives/C04HVBETC9Z) | [calendar-faas](https://groups.google.com/a/opentelemetry.io/g/calendar-faas) |  | [Austin Parker](https://github.com/austinlparker) | 
+| <a id="sig-faas"></a>Functions as a Service (FAAS) | Every other Wednesday at 7:30 PT | [Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc) | [#otel-faas](https://cloud-native.slack.com/archives/C04HVBETC9Z) | [calendar-faas](https://groups.google.com/a/opentelemetry.io/g/calendar-faas) |  | [Austin Parker](https://github.com/austinlparker) | 
```

Comment:
Please also make sure to update the calendar accordingly (or ask someone with permissions e.g. @trask or @mtwo to do so).

Reply:
I have permissions and I will update it.